src/services/textChanges.ts

```diff
@@ -1166,6 +1180,34 @@ namespace ts.textChanges {
                     Debug.assertNever(gp);
             }
         }
+
+        function getRangeToDeleteJsDocTag(node: JSDocTag, sourceFile: SourceFile): TextRange | undefined {
+            const { text } = sourceFile;
+            const { parent } = node;
+            if (parent.kind === SyntaxKind.JSDocComment && parent.comment === undefined && parent.tags!.length === 1) {
+                return createTextRangeFromNode(parent, sourceFile);
+            }
+
+            let pos = node.getStart(sourceFile);
+            Debug.assert(text.charCodeAt(pos) === CharacterCodes.at);
+            pos--;
+            while (isWhiteSpaceSingleLine(text.charCodeAt(pos))) pos--;
+
+            // If the line doesn't begin with `*`, give up.
+            if (text.charCodeAt(pos) !== CharacterCodes.asterisk) return undefined;

```

Comment:
So deletion fails if the line doesn’t start with asterisk? I don’t like that limitation but if you don’t want to fix it, can you document it with a test case?